html/create-user.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
`<nav>` in `<header>`

Reply:
Alright they should all be taken care of now